### PR TITLE
[FEATURE] Group-based content fetching in Frontend

### PR DESCRIPTION
**Pull request by linawolf**
resolves https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/922

RELEASES: main

### CODE REVIEW

File: Documentation/ContentObjects/Pageview/Index.rst [hunk 1]
```diff
@@ -158,6 +157,10 @@ Properties
         sub-property :typoscript:`options` can be used to pass parameters to the
         processor class.
 
+        It is recommended to use the  :ref:`PageContentFetchingProcessor <PageContentFetchingProcessor>`
```
Comment (froemken) [inline Documentation/ContentObjects/Pageview/Index.rst:160]: If it is recommended I prefer to extend the linked example accordingly

**Review by froemken — COMMENTED** [review]
Can you update the linked example?

File: Documentation/ContentObjects/Pageview/Index.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/ContentObjects/Pageview/Index.rst:160]: The example displayed here uses that data processor?

Comment (froemken) [reply Documentation/ContentObjects/Pageview/Index.rst:160]: This is the current original:
https://docs.typo3.org/m/typo3/reference-typoscript/main/en-us/ContentObjects/Pageview/Index.html#confval-pageview-dataprocessing

The linked example goes to here:
https://docs.typo3.org/m/typo3/reference-typoscript/main/en-us/ContentObjects/Pageview/Index.html#cobj-pageview-dataProcessing-example

This example does not contain the recommended DataProcessor.

What do you think? Keep it as it is? Update the linked example? Update the link to the better and new examples of your PR

Comment (linawolf) [reply Documentation/ContentObjects/Pageview/Index.rst:160]: That example does not yet include information on how to display the content elements. as displaying the content elements is incomplete without https://review.typo3.org/c/Packages/TYPO3.CMS/+/83718/2 beeing merged I would keep the other example as-is for now

**Comment by linawolf** [comment]
@froemken see my detailed explaination above: That example does not yet include information on how to display the content elements. as displaying the content elements is incomplete without https://review.typo3.org/c/Packages/TYPO3.CMS/+/83718/2 beeing merged I would keep the other example as-is for now

File: Documentation/DataProcessing/PageContentFetchingProcessor.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
+..  include:: /Includes.rst.txt
+..  _PageContentFetchingProcessor:
+
+============================
+PageContentFetchingProcessor
+============================
+
+This data processor loads all :sql:`tt_content` records from the current
```
Comment (sfroemkenjw) [inline Documentation/DataProcessing/PageContentFetchingProcessor.rst:8]:
```suggestion
This Data Processor loads all :sql:`tt_content` records from the current
```

File: Documentation/DataProcessing/_PageContentFetchingProcessor/_BackendLayout.tsconfig [hunk 3]
```diff
@@ -0,0 +1,41 @@
+mod.web_layout.BackendLayouts {
+    default {
```
Comment (sfroemkenjw) [inline Documentation/DataProcessing/_PageContentFetchingProcessor/_BackendLayout.tsconfig:2]: In TYPO3 we have a 2 letter indent: https://github.com/TYPO3/typo3/blob/main/.editorconfig#L51
Or do we need the 4 letter indent for better rst rending?

Comment (linawolf) [reply Documentation/DataProcessing/_PageContentFetchingProcessor/_BackendLayout.tsconfig:2]: Nope, you can fix the editorconfig in a follow up. I think 2 space indent is better anyway

**Review by froemken — APPROVED** [review]

**Pull request merged**